src/main/java/useCases/SettingsPage/ChangePassword.java

```diff
@@ -0,0 +1,8 @@
+package useCases.SettingsPage;
+
+
+public class ChangePassword {
+    public ChangePassword(String password) {
+
```

Comment:
Why does this method do nothing? If you plan on implementing it later it's fine though, but beware that anything calling this method is not going to do anything.